migrations/000048_AddRetries.up.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TYPE ImportFileStatus ADD VALUE 'RETRY';
```

Comment:
this shouldn't be needed... will look at rest of PR first

Reply:
I'm fine w/ this staying, but I think the extra state made the pr slightly more complex than it needed to be.
having the state be pending w/ retries>0 indicates the same thing.

no need to change

Reply:
removed.